docs/system/hardware-metrics.md

```diff
@@ -12,9 +12,7 @@ when creating instruments not explicitly defined in the specification.
 
 <!-- toc -->
 
-- [Common hardware attributes](#common-hardware-attributes)
```

Comment:
it should still be relevant for other metrics defined in this document

Reply:
This file will be removed. I have deleted this portion only for visibility on what is moved out

Reply:
In the meantime, while it is not removed, could you please add a link to the section in the new file so that someone looking at this file will know that there are some common attributes?